src/main/java/frc/robot/util/Utilities.java

```diff
@@ -27,4 +27,8 @@ public static double deadzone(double input) {
   public static double deadzone(double input, double deadzone) {
     return Math.abs(input) < Math.abs(deadzone) ? 0 : input;
   }
+
+  public static double clip(double input, double max, double min) {
```

Comment:
Nitpick: Change the order of the parameters to `double input, double min, double max`, it just makes more sense intuitively

Reply:
Resolved in 90cb14b